mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubSearch.kt

```diff
@@ -35,14 +35,18 @@ import kotlin.jvm.Throws
 /**
  * Allows to access GitHub Search API.
  */
-public interface GitHubClientService {
+public interface GitHubSearch {
 
     /**
      * Searches for user `Mention`s on GitHub.
      *
      * Mentions are searched by the name of the mentioned user. Mentions must be within items
      * updated between the `updateAfter` time and the present. The `PersonalAccessToken` is used
      * to access the GitHub API.
+     *
+     * @param username The name of the user whose mentions are to be fetched.
+     * @param token The `PersonalAccessToken` to access user's private repositories
+     * @param updatedAfter The time from which the mentions are searched.
      */
     @Throws(CannotFetchMentionsFromGitHubException::class)
     public fun fetchMentions(
```

Comment:
After you renamed the interface, this name does not play well with "...Search" anymore. Please  suggest another name here as well. Same concerns the name of the exception.